tests/app/celery/test_provider_tasks.py

```diff
@@ -115,15 +116,17 @@ def test_should_technical_error_and_not_retry_if_invalid_email(
     assert notification.status_reason == 'Email address is in invalid format'
 
 
+@pytest.mark.parametrize('exception', [NonRetryableException, InvalidPhoneError])
 def test_should_queue_callback_task_if_non_retryable_exception_is_thrown(
```

Comment:
This should probably be renamed because NonRetryableException isn't the only thing being tested.

Reply:
renamed